File: c/4-functions/workout/fw-3.c
#include <stdio.h>
#include <math.h>
int main()
{
    int a;
    long fact =1;
    scanf("%d",&a);
    if(a<0)
    {
        printf("Invalid input");
    }
    else
    {
        
         for(int i = 1;i<=a;i++)
         {
             fact*=i;
         }
         printf("%ld\n",fact);
         
    }
}

File: java/6-functions/workout/fw-3.java
import java.util.*;
public class A {
     public static double calculateArea(double r) {
        return Math.PI * r * r;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        if (sc.hasNextDouble()) {
            double radius = sc.nextDouble();
            if (radius >= 0) {
                double area = calculateArea(radius);
                System.out.printf("%.2f\n", area);
            } else {
                System.out.println("Invalid input");
            }
        } else {
            System.out.println("Invalid input");
        }
    }
   
}